src/ORM/Table.php

```diff
@@ -1175,6 +1175,32 @@ public function belongsToMany($associated, array $options = [])
      */
     public function find($type = 'all', $options = [])
     {
+        $deprecatedKeys = [
+            // 'fields', // 24 error warnings
+            // 'conditions', // 77 errors warnings, core seems to rely on thi...
+            'join', // 0 error warnings
+            // 'order', // 19 error warnings
+            // 'limit', // 21 error warnings
+            'offset', // 0 error warnings
+            // 'group', // 1 error warning in cakephp/src/Controller/Component/PaginatorComponent.php, line: 201
+            'having', // 0 error warnings
+            // 'contain', // 83 error warnings, core seems to rely on this...
+            // 'page', // 17 error warnings
+        ];
+
+        $deprecatedKeyWarns = [];
```

Comment:
Could be 'deprecatedKeys' 🚲 🏠

Reply:
Not really as that variable is being used above for the list to compare against ;-)